src/TuDa.CIMS.Api/Controllers/PurchaseController.cs

```diff
@@ -0,0 +1,98 @@
+﻿﻿using Microsoft.AspNetCore.Mvc;
+using TuDa.CIMS.Api.Interfaces;
+using TuDa.CIMS.Shared.Dtos;
+
+namespace TuDa.CIMS.Api.Controllers;
+
+[ApiController]
+[Route("api/purchase")]
```

Comment:
🔧 Please use the plural of purchase for routes
```suggestion
[Route("api/purchases")]
```